res/controllers/Intech TEK2.scripts.js

```diff
@@ -0,0 +1,32 @@
+const TEK2 = {};
```

Comment:
Have you ensured that this still works with `const`? I'm skeptical that it will work because `const` (and `let`) work a little differently than `var`.

Reply:
I checked it, and it's only works with var. but pre-commit is not let me use var. why is const or let not working? "TEK2 is not defined"

Reply:
its a little complicated on why it makes a difference. In short `var` modifies the `this` object of the current scope while `let` and `const` don't. Since we need lookup `TEK2` on the global object when handling midi message, its important to use `var`. So just do 
```suggestion
// eslint-disable-next-line no-var
var TEK2 = {};
```
instead.

Reply:
thank you!